src/containers/pages/JurisdictionAssignment/EntryView/utils.tsx

```diff
@@ -0,0 +1,131 @@
+import { Result } from '@onaio/utils/dist/types/types';
```

Comment:
I think this utils module needs docstrings and significant documentation to explain what is happening